src/nodes.rs

```diff
@@ -175,6 +175,14 @@ impl UserDefinedLogicalNode for SeafowlExtensionNode {
         Arc::from(self.clone())
     }
 
+    fn with_exprs_and_inputs(
+        &self,
+        _exprs: Vec<Expr>,
+        _inputs: Vec<LogicalPlan>,
+    ) -> datafusion_common::Result<Arc<dyn UserDefinedLogicalNode>> {
+        todo!("Should we implement this")
```

Comment:
Yeah, looks like it's the same thing as `from_template`, so use that impl? I'm surprised we don't crash with this `todo()`.

https://github.com/apache/datafusion/blob/5bfc11ba4ac4f11eaf9793c668e4a064fb697e6e/datafusion/expr/src/logical_plan/extension.rs#L79-L103

```suggestion
        Ok(Arc::from(self.clone()))
```